<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html lang=en>

<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta http-equiv="Content-Style-Type" content="text/css">
<meta name="author" content="Yuki Naito et al.">
<title>siDirect help</title>
<style type="text/css">
<!--
/* font */
	* { font-family:verdana,arial,helvetica,sans-serif }
	p,ul,ol,h4 { font-size:8pt }
	.u { background-color:#FF99FF }
	.r { background-color:#66FFFF }
	.a { background-color:#99FF00 }
/* hyperlink */
	a:link,a:visited {
		text-decoration:none;
		color:#004080;
	}
	a:hover {
		text-decoration:none;
		color:red;
	}
-->
</style>
</head>

<body>

<a href=".."><img src="ocean.jpg" height=80 width="100%" border=0></a>
<div style="font-size:8pt">
<a style="color:black" href="."><font size=5>siDirect </font><font size=4>version 2.1<small>&beta;</small> </font></a>help page.
</div>

<hr><!-- __________________________________________________ -->


<h3>About siDirect</h3>

<p>siDirect is a web server that provides efficient and target-specific siRNA design for mammalian RNAi. In version 2.0 (2009), the siRNA design algorithm was extensively updated to reduce off-target silencing effects. This update was based on our finding that the capability of siRNAs to induce off-target effects is highly correlated with the thermodynamic stability of the 'seed' duplex (<a href="https://doi.org/10.1093/nar/gkn902">reference 6</a>). The updated siRNA design algorithm, incorporating these mechanistic features, is combined with the previously reported reliable algorithms.</p>

<h3>Examples</h3>

<h4>Example 1 | Designing siRNAs for human claudin 17 (CLDN17)</h4>

<ol>
	<li>Enter the accession number for human claudin 17 (NM_012131).
	<li>Click 'retrieve sequence' to obtain the nucleotide sequence from GenBank.
	<li>Alternatively, you can directly paste a nucleotide sequence.<br>
		Accepted input types are FASTA or a plain nucleotide sequence up to 10 kbp.
	<li>Click 'design siRNA'.
</ol>

<img src="CLDN17_top.gif">

<ol start=5>
	<li>Results:
	<ol type=A>
		<li>siRNA target positions.
		<li>siRNA target sequences (21nt + 2nt overhang) with links to the off-target list.
		<li>siRNA efficacy predictions. siRNAs with <font class=u>U</font>, <font class=r>R</font>, <font class=a>A</font> satisfy the functional siRNA design algorithms of <font class=u>U</font>i-Tei <i>et al.</i> (<a href="https://doi.org/10.1093/nar/gkh247">reference 3</a>), <font class=r>R</font>eynolds <i>et al.</i> (<a href="https://doi.org/10.1038/nbt936">reference 4</a>), and <font class=a>A</font>marzguioui <i>et al.</i> (<a href="https://doi.org/10.1016/j.bbrc.2004.02.157">reference 5</a>), respectively.
		<li>Sequence similarity search results against human mRNAs. The number of hits with a complete match (19/19 matches), one mismatch (18/19 matches), two mismatches (17/19 matches), or three mismatches (16/19 matches) is shown. A number '1' in the 0(+) column usually indicates a complete match against the intended target mRNA (<i>e.g.</i>, claudin 17).
		<li>Calculated Tm of the siRNA seed region. Selecting an siRNA with a lower seed Tm reduces off-target effects. &rarr; Ui-Tei <i>et al.</i> (<a href="https://doi.org/10.1093/nar/gkn902">reference 6</a>)
		<li>Graphical view of designed siRNAs. Click each siRNA to show the off-target list.
		<li>Tab-delimited siRNA list. You can copy-paste the result into Excel or text editors, etc. 
	</ol>
</ol>

<img src="CLDN17_result.gif">

<ol start=6>
	<li>siRNA information.
	<li>List of off-target candidates for individual siRNAs. The alignment between each off-target candidate and the siRNA sequence clarifies the locations of mismatches. Hits with a complete match (19/19 matches), one mismatch (18/19 matches), two mismatches (17/19 matches), or three mismatches (16/19 matches) are shown.
	<ol type=A>
	</ol>
</ol>

<img src="CLDN17_offlist.gif">

<ol start=8>
	<li>Design sense and antisense RNA oligonucleotides from the siDirect results.
</ol>

<img src="target2si.svg" width=360 height=115 border=0>

<h4>Example 2 | Designing shRNA vectors (with pol III promoter) for human vimentin (VIM)</h4>

<ol>
	<li>Enter the accession number for human vimentin (NM_003380).
	<li>Click 'retrieve sequence' to obtain the nucleotide sequence from GenBank.
	<li>Alternatively, you can directly paste a nucleotide sequence.<br>
		Accepted input types are FASTA or a plain nucleotide sequence up to 10 kbp.
	<li>Click 'Options: [click here]'.
	<li>Check 'Avoid contiguous A's or T's 4 nt or more'.<br>
		This prevents transcription termination by pol III promoters.
	<li>Check 'Custom pattern' and input 'NNGNNNNNNNNNNNNNNNNNNNN' in the field.<br>
		This fixes the third letter of the target sequence (<i>i.e.</i>, shRNA transcription start) to 'G'.
	<li>Check 'Only show siRNAs that match all checked criteria'.
	<li>Click 'design siRNA'.
</ol>

<img src="VIM_options.gif">

<ol start=9>
	<li>Design oligonucleotide sequence from the siDirect result.
</ol>

<img src="VIM_result.gif"><br><br>
<img src="target2sh.svg" width=597 height=240 border=0>

<h3>Methods</h3>

<p>
The diagram below illustrates the siRNA selection process in siDirect version 2. All possible 23-mer subsequences, corresponding to the complementary sequence of the 21-nt guide strand and the 2-nt 3&prime; overhang of the passenger strand within the target sequence, are generated and filtered in three selection steps.
</p>

<img src="bmc-fig1_r1.svg" width=600 height=405 border=0>

<h4>Step 1: Selection of highly functional siRNAs</h4>

<p>
In the first step, highly functional siRNA sequences are selected using our algorithm (<a href="https://doi.org/10.1093/nar/gkh247">reference 3</a>). We have revealed that efficient RNAi can be induced when the siRNA satisfies the following three sequence conditions simultaneously: A/U at the 5&prime; terminus of the guide strand; G/C at the 5&prime; terminus of the passenger strand; and a minimum of 4 A/U residues within the 5&prime; terminal 7 bp of the guide strand. Additionally, the presence of a G/C stretch longer than 9 bp should be avoided. The proportion of functional siRNA sequences selected by this algorithm is 14.7% of all 23-mer sequences derived from RefSeq 30 human transcripts.
</p>

<h4>Step 2: Reduction of seed-dependent off-target effects</h4>

<p>
We have found a significant correlation between off-target effects and the thermodynamic stability (Tm) of the seed-target duplex formed between the nucleotides positioned at 2-8 from the 5&prime; end of the siRNA guide strand and its target sequence (<a href="https://doi.org/10.1093/nar/gkn902">reference 6</a>). In the second step, to avoid off-target effects, the Tm for the seed-target duplex is calculated using the nearest neighbor model and the thermodynamic parameters for RNA duplex formation. As shown in our previous report, a calculated Tm of 21.5 ºC serves as a benchmark to distinguish nearly off-target-free seed sequences from the off-target-positive ones, thus being used as the initial standard. Moreover, it has been revealed that RNAi silencing is occasionally induced by passenger strands of functional siRNAs, which also participate in seed-dependent off-target gene silencing. Thus, siRNAs with a seed-target Tm below 21.5 ºC for both the guide and passenger strands are selected. As a result, 3.0% of all human 23-mer sequences remain available. The calculated Tm value for each siRNA is provided on the siDirect result page.
</p>

<h4>Step 3: Elimination of near-perfect matched genes</h4>

<p>
Several studies have indicated that the impact of single-base mismatches between the siRNA guide strand and the target mRNA varies, depending on the positions of the mismatch and/or the sequence of the siRNA. However, it is obvious that even when the Tm value of the seed-target duplex is sufficiently low, target gene silencing can still occur if the non-seed region is completely complementary. Therefore, in the third step, siRNAs that possess near-perfect matches to any other non-targeted transcripts are excluded. In siDirect version 2, off-target searches are conducted for 19-mer sequences at positions 2-20 of both strands of the siRNA duplex, because these 19 nucleotides are thought to be involved in target mRNA recognition. Since widely-used BLAST algorithms frequently overlook near-perfect match candidates, siDirect utilizes our fast and sensitive algorithm (<a href="https://doi.org/10.1093/bioinformatics/bti155">reference 7</a> and <a href="https://gggenome.dbcls.jp/">GGGenome</a>). Additionally, all near-perfect match hits are precomputed for all functional human siRNAs and stored in the SQLite database to enhance computational efficiency. This enables siDirect to generate a list of siRNA candidates within seconds. The output page provides the minimum number of mismatches against any near-perfect match candidates for each siRNA. Clicking on an individual siRNA displays a detailed list of candidate genes. By default, siRNA sequences with at least two mismatches to any other non-targeted transcripts are selected.
</p>

<h3>Previous releases</h3>

<ul>
	<li><a href="https://siDirect2.RNAi.jp/v2.0/">
		siDirect v2.0 web server</a> (Nov, 2009)
	<li>siDirect web server (May, 2005) [discontinued]
	<li>siDirect web server (May, 2004) [discontinued]
</ul>

<h3>Useful tools</h3>

<ul>
	<li><b>GGGenome</b>: ultrafast DNA sequence search engine maintained by our group.<br>
		<a href="https://GGGenome.dbcls.jp/">https://GGGenome.dbcls.jp/</a>
	<li><b>CRISPRdirect</b>: target-specific CRISPR guide RNA design software maintained by our group.<br>
		<a href="https://crispr.dbcls.jp/">https://crispr.dbcls.jp/</a>
</ul>

<h3>References</h3>

<h4>siDirect web server</h4>
<p>If you use siDirect in your work, please cite references 1 and 2.</p>
<ol>
	<li>Naito Y, Yoshimura J, Morishita S, Ui-Tei K. (2009)<br>
		<a href="https://doi.org/10.1186/1471-2105-10-392">
		siDirect 2.0: updated software for designing functional siRNA with reduced seed-dependent off-target effect.</a><br>
		<i>BMC Bioinformatics</i> <b>10</b>, 392.
	<li>Naito Y, Yamada T, Ui-Tei K, Morishita S, Saigo K. (2004)<br>
		<a href="https://doi.org/10.1093/nar/gkh442">
		siDirect: highly effective, target-specific siRNA design software for mammalian RNA interference.</a><br>
		<i>Nucleic Acids Res.</i>, <b>32</b>, W124-W129.
</ol>

<h4>Designing functional siRNAs</h4>
<ol start=3>
	<li>Ui-Tei K, Naito Y, Takahashi F, Haraguchi T, Ohki-Hamazaki H, Juni A, Ueda R, Saigo K. (2004)<br>
		<a href="https://doi.org/10.1093/nar/gkh247">
		Guidelines for the selection of highly effective siRNA sequences for mammalian and chick RNA interference.</a><br>
		<i>Nucleic Acids Res.</i>, <b>32</b>, 936-948.
	<li>Reynolds A, Leake D, Boese Q, Scaringe S, Marshall WS, Khvorova A. (2004)<br>
		<a href="https://doi.org/10.1038/nbt936">
		Rational siRNA design for RNA interference.</a><br>
		<i>Nat. Biotechnol.</i>, <b>22</b>, 326-330.
	<li>Amarzguioui M, Prydz H. (2004)<br>
		<a href="https://doi.org/10.1016/j.bbrc.2004.02.157">
		An algorithm for selection of functional siRNA sequences.</a><br>
		<i>Biochem. Biophys. Res. Commun.</i>, <b>316</b>, 1050-1058.
</ol>

<h4>Reducing off-target effects</h4>
<ol start=6>
	<li>Ui-Tei K, Naito Y, Nishi K, Juni A, Saigo K. (2008)<br>
		<a href="https://doi.org/10.1093/nar/gkn902">
		Thermodynamic stability and Watson-Crick base pairing in the seed duplex are major determinants of the efficiency of the siRNA-based off-target effect.</a><br>
		<i>Nucleic Acids Res.</i> <b>36</b>, 7100-7109.
	<li>Yamada T, Morishita S. (2005)<br>
		<a href="https://doi.org/10.1093/bioinformatics/bti155">
		Accelerated off-target search algorithm for siRNA.</a><br>
		<i>Bioinformatics</i>, <b>21</b>, 1316-1324.
</ol>

<h3>Licenses</h3>

<p>
Researchers from academic organizations are eligible to use siDirect freely for academic purposes. However, if users from academic, non-profit, or commercial organizations design or utilize siDirect for any commercial purposes, such as including it in a commercial product sake or profits research, they are required to make licensing agreements by contacting license@rnai.jp.
</p>

<hr><!-- __________________________________________________ -->

<p><font color=gray>siDirect v2.1&beta; | Last modified on Sep 5, 2023.</font></p>

</body>
</html>
